src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceDateInput.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <label>
+    {{ label }}
+    <v-date-input
+      :model-value="new Date(modelValue)"
+      :value="formattedDate"
+      :aria-label="label"
+      prepend-icon=""
+      label=""
```

Comment:
tested this on vuetify playgound, I dont think we need to define label as an empty string to get rid of it. 

https://play.vuetifyjs.com/#eNpVjsEKwjAMhl+l5GztXerA57Ae6hqhEGOo7S7iu1uzDbZL+PPx8SfXD7zL6C4ix6khnMBXfArFikNgY/xko4hGXcYX15gZy4IUpm7bzNKqkYKCnGzu4jlAAEPxjqRx8G7rrqUd7lv/ZD7q3eaZxgkfXUqB4XvgRqTj9gM48EEj